requirements.txt

```diff
@@ -32,6 +32,8 @@ cmake
 Sphinx >= 3.5.4
 sphinx-rtd-theme >= 0.5.2
 pip-licenses
+pydata-sphinx-theme==0.9.0
+sphinx-design
```

Comment:
what is using this?

Reply:
This is for the grid "cards" on the new landing page.

Reply:
https://pydata-sphinx-theme.readthedocs.io/en/stable/user_guide/web-components.html